bears/haskell/HaskellLintBear.py

```diff
@@ -38,11 +38,15 @@ def process_output(self, output, filename, file):
         output = json.loads(output)
 
         for issue in output:
-            assert issue['startLine'] == issue['endLine']
             diff = Diff(file)
+            from_lines = issue['from'].splitlines()
+            to_lines = issue['to'].splitlines()
```

Comment:
can't those fail? because when `issue['to']` or `issue['from']` is `None` like in the exception in the issue, it will fail because there's no `splitlines()` in `None`^^